assignments/week_1/exercise_1_1/exercise_1_1.md

```diff
@@ -0,0 +1,163 @@
+  ```
+  
+</details>
+
+2. Set each float in wheel_speeds to 0 using square brackets ```[]``` to access each index
+<details>
+  <summary>Answer</summary>
+  
+  ```c++
+  wheel_speeds[0] = 0;
+  wheel_speeds[1] = 0;
+  wheel_speeds[2] = 0;
+  wheel_speeds[3] = 0;
+  ```
+  
+</details>
+
+3. In the while loop create a set else if statements for each case. Remember to include one to check if direction is ‘w’,’a’,’s’, ’d’,‘e’ or none of those things. You can just use ```==``` to check if direction is equal to a certain character.
+<details>
+  <summary>Answer</summary>
+  
+  Note that the order of what you check does not matter, as long as the if statement is frist, the else statement is last and the else ifs are in between.
+  
+  ```c++
+  if (direction == ‘w’) {}
+  else if (direction == ‘a’) {}
+  else if (direction == ‘s’) {}
+  else if (direction == ‘d’) {}
+  else if (direction == ‘e’) {}
+  else {}
+  ```
+  
+</details>
+
+4. Within the else if statement for ‘w’, create a for loop which sets each float in wheel_speeds to 1. The for loop should count from 0 to 3 and use the counter with square brackets as the index for wheel_speeds.
+<details>
+  <summary>Answer</summary>
+  
+  ```c++
+  for (int i = 0; i < 4; i++) {
+    wheel_speeds[i] = 1; 
+  }
+  ```
+  
+</details>
+
+5. Within the else if statement for ‘a’, set the floats at indices 0 and 2 to 0.5 and set the floats at indices 1 and 3 to 1 in the wheel_speeds vector. Do this by individually accessing each index using square brackets ```[]```.
+<details>
+  <summary>Answer</summary>
+
+  ```c++
+  wheel_speeds[0] = 0.5;
+  wheel_speeds[2] = 0.5;
+  wheel_speeds[1] = 1;
+  wheel_speeds[3] = 1;
+  ```
+  
+</details>
+
+6. Within the else if statement for ‘s’, create a for loop which sets each float in wheel_speeds to -1.
+<details>
+  <summary>Answer</summary>
+
+  ```c++
+  for (int i = 0; i < 4; i++) {
+    wheel_speeds[i] = -1;
+  }
+  ```
+  
+</details>
+
+7. Within the if else statement for ‘d’, set the floats at indices 1 and 3 to 0.5 and set the floats at indices 0 and 2 to 0.5 in the wheel_speeds vector.
+<details>
+  <summary>Answer</summary>
+
+  ```c++
+  wheel_speeds[0] = 1;
+  wheel_speeds[2] = 1;
+  wheel_speeds[1] = 0.5;
+  wheel_speeds[3] = 0.5;
+  ```
+  
+</details>
+
+8. Within the if else statement for ‘e’, create a for loop which sets all wheel speeds to 0, then create a print statement that prints “Shutting Down” and set the the keep_going boolean to false to stop the while loop from continuing.
+<details>
+  <summary>Answer</summary>
+
+  ```c++
+  for (int i = 0; i < 4; i++) {
+    wheel_speeds[i] = 0;
+  }
+  std::cout << "Shutting down\n";
+  keep_going = false;
+  ```
+  
+</details>
+
+9. Within the else statement create a print statement which prints “Sorry I didn’t get that, try again” on a line by itself.
+<details>
+  <summary>Answer</summary>
+
+  ```c++
+  std::cout << "Sorry I didn't get that, try again" << std::endl;
```

Comment:
This should be updated to match the solution code.
```suggestion
9. Within the else statement create a print statement which prints “Sorry, I didn’t get that. Try again.” on a line by itself.
<details>
  <summary>Answer</summary>

  ```c++
  std::cout << "Sorry, I didn't get that.  Try again." << std::endl;
```